### PR TITLE
reporting-server:Created script and cronjob to delete logs from database > 7 days

### LINKED ISSUE #399: reporting-server: Create a cronjob that runs a query to delete data > 7 days
## Feature request

Remove data >7 days from the reporting-database

### Description

One of the specifications was to remove data from the reporting database if they're older than 7 days. One way of doing that is using a Kubernetes cronjob that calls a query to clean the database.
  
https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/

### PR DESCRIPTION
**Pull request by matiasbavera**
Signed-off-by: Matias Bavera <matiasbavera@gmail.com>

fixes #399 

## What's new
Created script to delete logs from database > 7 days. This script is built into the application.
To run it you need to activate the virtual environment and run `reporting_server_clean_logs`.

## Self-checks

- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

### CODE REVIEW

File: packages/reporting-server/setup.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
     entry_points={
         "console_scripts": [
             "reporting_server=rest_server.__main__:main",
+            "clean_logs=rest_server.clean_logs:main",
```
Comment (koonpeng) [inline packages/reporting-server/setup.py:34]: Can we name this to be more specific? `clean_logs` doesn't really imply that it is an auxiliary tool for the reporting server. Maybe add a prefix to the name, have a combined script that expose this function and other future functions as sub commands, or integrate this into the main script with a sub command.

File: example-deployment/README.md [hunk 2]
```diff
@@ -350,6 +350,14 @@ deploy it
 .bin/minikube kubectl -- apply -f k8s/reporting.yaml
 ```
 
+## CronJobs
+
+deploy cronjobs
```
Comment (koonpeng) [inline example-deployment/README.md:355]: Should we have an example on how to configure the job schedule, maybe a link to relevant docs in kubernetes and/or cron format.

Comment (matiasbavera) [reply example-deployment/README.md:355]: Yeah! that's a good idea! Thanks!

File: packages/reporting-server/setup.py (hunk 1 above)
Comment (matiasbavera) [reply packages/reporting-server/setup.py:34]: I added this `reporting_server_clean_logs` for now. And for the future, if I understand this right, do you want me to create something like: `reporting_server_script --clean_logs`,  `reporting_server_script --migration`, etc?

Comment (koonpeng) [reply packages/reporting-server/setup.py]: I think it's fine as long as one can easily identify that the script is meant for reporting server. A name like `clean_logs` can be quite vague as to what logs it's cleaning. So either a prefix on all of the reporting server scripts, or a meta-script that combines the functionalities of all the other scripts.

**Review by koonpeng — APPROVED** [review]
